package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "firebase-tools",
-  "version": "11.6.0",
+  "version": "11.7.0",
```

Comment:
Make sure to fix up issues like this and the CHANGELOG removing everything. This in particular is a dangerous inclusion.